tools/operator/README.md

```diff
@@ -0,0 +1,107 @@
+# Fabric Network Operator
+
+- A tool to launch fabric network on kubernetes cluster or local machine with a docker-compose
+file using a network input file and gives back connection profiles for each peer organization
+to use with any client. This tool also uses a testclient to perform the fabric operations which
+includes creating a channel, joining peers to a channel, anchor peer update, installing, upgrading
+and instantiating the chaincode, performing invokes and queries, based on a test input file. This tool is also used for creating channelTx, migrating the network to etcdraft, checking the health of peers and orderers and upgrading the network on providing a specific network input file. This uses `ytt` 
```

Comment:
This provides lots of helpful information. I would like to reword just a bit for readability:

file. This tool creates the network based on a network specification file, and also creates connection profiles for each peer organization, which can be used by any client. It also can use its own testclient to generate channelTx transaction files, and perform fabric operations defined as "actions" in a test input file, such as creating channels, joining peers to a channel, anchor peer updates, installing, upgrading
and instantiating chaincodes, and performing invokes and queries, migrating a network from kafka to etcdraft, checking the health of peers and orderers, and upgrading the network.

Reply:
Done